packages/boot/test/upgrading/upgrade-contracts.test.js

```diff
@@ -13,8 +13,8 @@ import { buildVatController } from '@agoric/swingset-vat';
 const test = anyTest;
 
 const bfile = name => new URL(name, import.meta.url).pathname;
-const importSpec = spec =>
-  importMetaResolve(spec, import.meta.url).then(u => new URL(u).pathname);
+const importSpec = async spec =>
+  new URL(importMetaResolve(spec, import.meta.url)).pathname;
```

Comment:
Not necessary to fix everything in one pass, but notably, this is an opportunity to replace `.pathname` with `fileURLToPath` as that’s more robust across platforms or checkout directories with spaces (`.pathname` will have `%20` instead of the obligate space).